benchmarks/src/bin/tpch.rs

```diff
@@ -396,7 +396,8 @@ async fn get_table(
             }
             "parquet" => {
                 let path = format!("{}/{}", path, table);
-                let format = ParquetFormat::default().with_enable_pruning(true);
+                let format = ParquetFormat::new(ctx.config_options())
```

Comment:
As the parquet format now reads defaults from `ConfigOptions` they need to be passed to the constructor